### ISSUE TITLE
Global ActionBar showing

### ISSUE THREAD

**Issue by kspearrin** [keerl/tab-drawer-nav-alt-ng#1]
Thanks for the example. Any idea why I am getting this "global" action bar at the top of my app with the app name? Causes two to be stacked. I only want the bottom one which is defined per page like in your example. I can't find where this global one is coming from, but it shows on all pages.

![image](https://user-images.githubusercontent.com/1190944/46870766-90873480-cdfd-11e8-956b-3fd078065cd3.png)

**Comment by keerl** [comment]
In your `tabs.component.ts` file you can hide the action bar there. That can be done in the constructor like this: 
```
constructor(private page: Page) {
        this.page.actionBarHidden = true;
}
```

**Comment by kspearrin** [comment]
Any idea why you didn't have to do that on your example? I don't see that anywhere here.

**Comment by keerl** [comment]
Actually when I add what I suggested before it actually gets rid of the action bar I want. I'm not sure why the top one is showing for you. Does it behave like that when you clone my project exactly?

**Comment by keerl** [comment]
From a quick glance it looks like your Vault component isn't really doing anything and is acting as another container for the pages (which maybe could cause the double action bar). What if you just got rid of it so your routes look something like this (have not tested):

```
const routes: Routes = [
    { path: '', redirectTo: '/tabs', pathMatch: 'full' },
    {
        path: 'tabs',
        component: TabsComponent,
        children: [
            { path: 'groupings', component: GroupingsComponent },
            { path: 'ciphers/:id', component: CiphersComponent }  
        ],
    },
];
```

**Comment by keerl** [comment]
I am able to recreate the issue, I'll have to take a deeper look at things. My demo did not behave like this so I have to see if I can find the difference.

**Comment by kspearrin** [comment]
Recreate it using my repo or yours?

**Comment by keerl** [comment]
In mine. I actually lied, the same problem appears in my demo. The Android button kills the app, but if you do back() it will work. Which also means the back navigation will not work naturally on iOS. I will have to look further into this (which may not be quick). Only solution I can think right now is overriding the Android button to trigger router.back() and on iOS you will have to make a "fake" back button to trigger the router.back().

**Comment by keerl** [comment]
Or you can just have your details page go to another full page instead of in the tabs.

**Comment by kspearrin** [comment]
It works if you nest another `<page-router-outlet></page-router-outlet>` like I have now with my VaultComponent, but you just have to hide the extra actionbar.

**Comment by keerl** [comment]
Interesting. Seems to break the navigation to an outside page though. Trying to see if I can fix that.

**Comment by kspearrin** [comment]
Yes, it seems that trying to navigate to anything that isn't in the tabs route itself won't work (like trying to go back to login page after logging out for example). Let me know if you come up with any ideas.
